exercises/ruby-2-a/ruby_2_a.rb

```diff
@@ -1 +1,19 @@
-TODO
\ No newline at end of file
+# frozen_string_literal: true
```

Comment:
The second part of the research experiment has its starting points (of which should be multiple) in a different location: `.meta/start/{1,2,3}/ruby_2_a.rb`.  This means that this file can be removed and that the [example implementations to part 1](https://github.com/exercism/research_experiment_1/tree/master/exercises/ruby-1-a) should be used as  `.meta/start/1/ruby_2_a.rb`, `.meta/start/2/ruby_2_a.rb` and `.meta/start/3/ruby_2_a.rb`. To these files, the `scan_coupon` stub should be added, similar to how it is defined below.

See the [csharp-2-b example](https://github.com/exercism/research_experiment_1/tree/master/exercises/csharp-2-b/.meta/start). 

Let me know if this is not clear enough.

Reply:
For the second part I only have one example. Do we need one example for each start?

Reply:
No, one example is fine!